vagrant/boxes/Vagrantfile_rawhide_selinux

```diff
@@ -27,7 +27,7 @@ Vagrant.configure("2") do |config|
     dnf upgrade --refresh -y
 
     # Install build & test dependencies
-    dnf install -y attr busybox cryptsetup dosfstools fedpkg git nc qemu-kvm rpm-build rpmdevtools rust socat strace tpm2-tss-devel
+    dnf install -y attr busybox cryptsetup dosfstools fedpkg git nc qemu-kvm rpm-build rpmdevtools rust socat strace tpm2-tss-devel python3dist(jinja2)
```

Comment:
This needs to be in single quotes, to avoid shell expansion:

```suggestion
    dnf install -y attr busybox cryptsetup dosfstools fedpkg git nc qemu-kvm rpm-build rpmdevtools rust socat strace tpm2-tss-devel 'python3dist(jinja2)'
```